pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools>=51.1.1", "wheel", "setuptools_scm[toml]>=5.0"]
+requires = ["setuptools>=51.1.1", "wheel", "setuptools_scm[toml]>=5.0,<8"]
```

Comment:
It seems like setuptools-scm 8.0.1 no longer creates the version file. It is possibly a bug though.
I am suspecting it was introduced by this PR: https://github.com/pypa/setuptools_scm/pull/870

Note: I have tried renaming to the new config key `version_file` but that also did not work.

For now, we have to pin to `<8` until we can figure out why or how to fix this.

Reply:
Yeah, vyper ran into this too